zmscitizenapi/public/doc/partials/definitions.yaml

```
@@ -0,0 +1,33 @@
+  officesAndServices:
+    $ref: "schema/citizenapi/collections/officeServiceAndRelationList.json"
+  officesByService:
+    $ref: "schema/citizenapi/collections/officeList.json"
+  scope:
+    $ref: "schema/citizenapi/thinnedScope.json"
+  scopes:
+    $ref: "schema/citizenapi/collections/thinnedScopeList.json"
+  services:
+    $ref: "schema/citizenapi/collections/serviceList.json"
+  servicesByOffice:
+    $ref: "schema/citizenapi/collections/serviceList.json"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)